environment.go

```diff
@@ -216,6 +232,41 @@ func (e *envState) createQueues(ctx context.Context) error {
 	if err = e.notificationsQueue.SetRunner(runner); err != nil {
 		return errors.Wrap(err, "failed to set notifications queue runner")
 	}
+	rootSenders := []send.Sender{}
+	for _, s := range e.senders {
+		rootSenders = append(rootSenders, s)
+	}
+
+	e.closers["background-local-queue"] = func(ctx context.Context) error {
+		if !amboy.WaitCtxInterval(ctx, e.localQueue, 10*time.Millisecond) {
```

Comment:
Time should be a constant. Also, jobs could be longer because they might need to round trip to some API. Is 10 milliseconds too short?